Example/Backpack/View Controllers/GradientViewController.swift

```diff
@@ -20,20 +20,17 @@ import UIKit
 import Backpack
 
 enum GradientType {
-    case primary
     case baselinesSrim
 }
 
 class GradientViewController: UIViewController {
     @IBOutlet weak var gradientView: Backpack.GradientView!
-    var gradientType: GradientType = .primary
+    var gradientType: GradientType = .baselinesSrim
```

Comment:
Is this a typo? `Srim` seems weird

Reply:
That's exactly what it looks like. Fixed now